### PR TITLE
Added Wazuh Agent 4.2.5

### PR DESCRIPTION
**Pull request by sootysec**
Signed-off-by: Daniel Sutton <daniel@ducksecops.uk>

- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36305)

### CODE REVIEW

File: manifests/w/Wazuh/WazuhAgent/4.2.5/Wazuh.WazuhAgent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Wazuh.WazuhAgent
+PackageVersion: 4.2.5
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://packages.wazuh.com/4.x/windows/wazuh-agent-4.2.5-1.msi
+  InstallerSha256: 3C42ACC47D7B0DF07D25884BB5093FA64B0625B4BB9D2D8198AEB4570B4903B6
+  ProductCode: '{DC32A5AC-7AEC-4A24-AB7B-3C789CD379E2}'
```
Comment (ItzLevvie) [inline manifests/w/Wazuh/WazuhAgent/4.2.5/Wazuh.WazuhAgent.installer.yaml:12]:
```suggestion
  ProductCode: '{B874D271-7B8F-4F6A-8075-0265B962B8CD}'
```
wingetcreate or YamlCreate.ps1 should automatically update this for you.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Only one package per PR.

It seems like you added `Tenable Nessus` to this PR when it's supposed to be `Wazuh Agent`.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will need to remove `Tenable Nessus` from this PR and keep the `Wazuh Agent` package.

Only one package per PR is allowed.

**Comment by ducksecops** [comment]
@ItzLevvie sorry mistake on my part. Managed to add to wrong branch. I've removed the files now from this commit.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**